docs/guide/getting-started.md

```diff
@@ -71,10 +95,21 @@ Add some scripts to `package.json`.
 
 Serve the documentation site in the local server.
 
-```sh
+::: code-group
+
+```sh [npm]
+$ npm run docs:dev
+```
+
+```sh [yarn]
 $ yarn docs:dev
 ```
 
+```sh [pnpm]
+$ pnpm docs:dev
```

Comment:
I think it would be safer to add `run` argument here?

```suggestion
$ pnpm run docs:dev
```

Reply:
Yeah,it would be safer

Reply:
👍